packages/lib-classifier/src/store/RootStore.js

```diff
@@ -49,6 +54,16 @@ const RootStore = types
 
   .actions(self => {
     // Private methods
+    function _observeWorkInProgress() {
```

Comment:
Maybe this should be `_observeActiveSubject` since it observes `subjects.active`. But then it really observes `subjects.active?.stepHistory.checkForProgress` so IDK.

Reply:
Maybe `_observeCheckForProgress`? `_onProgressChange`?